xwiki-platform-core/xwiki-platform-notifications/xwiki-platform-notifications-test/xwiki-platform-notifications-test-tests/src/test/it/org/xwiki/platform/notifications/test/ui/NotificationsTest.java

```diff
@@ -122,11 +145,96 @@ public void testNotifications() throws Exception
         // Delete the "Deletion test page" and test the notification
         getUtil().login(FIRST_USER_NAME, FIRST_USER_PASSWORD);
         getUtil().deletePage(getTestClassName(), "DTP");
-        
+
```

Comment:
As said here : http://dev.xwiki.org/xwiki/bin/view/Community/Testing#HFunctionalTesting, it might be better to keep this into a single method, in order to limit performance issues (we talked about it with Vincent).